app/views/migration/new.html.mustache

```diff
@@ -1,4 +1,4 @@
 {{>atoms/meta/_head}}
 {{>layouts/css}}
```

Comment:
This can be removed, and the file app/views/layouts/css.html.mustache deleted.

Reply:
Done